commands/proposal_submission.go

```diff
@@ -251,6 +251,12 @@ func checkNewMarketChanges(change *types.ProposalTerms_NewMarket) Errors {
 		errs.AddForProperty("proposal_submission.terms.change.new_market.changes.decimal_places", ErrMustBeLessThan150)
 	}
 
+	if changes.PositionDecimalPlaces < 0 {
+		errs.AddForProperty("proposal_submission.terms.change.new_market.changes.position_decimal_places", ErrMustBePositiveOrZero)
+	} else if changes.PositionDecimalPlaces >= 150 {
```

Comment:
I think that should be limited to 6 isn't it?

Reply:
spec doesn't say I don't think unless I missed it, but I'm happy with whatever is agreed, I just kept it at the same accuracy as price for now.